app/controllers/api/v1/users/sessions_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Api::V1::Users::SessionsController < Devise::SessionsController
+  before_action :sign_in_params, only: :create
+  before_action :load_user, only: :create
+  protect_from_forgery with: :null_session
+
+  def create
+    if @user.valid_password?(sign_in_params[:password])
+      sign_in @user
+      render json: {
+        messages: "Signed in",
+        is_success: true,
+        data: { user: @user }
+      }, status: :ok
+    else
+      render json: { messages: 'Invalid email or password' }, status: :unauthorized
```

Comment:
Dado que el `#valid_password?` es un caso súper particular, no creo que sea necesario levantar una excepción y meterla al concern por ahora. 

Eso si, podrías usar el `respond_api_error` definido en el mismo concern para que quede estandarizado.

```suggestion
      respond_api_error(:unauthorized, { message: 'Invalid email or password' })
```

(recuerda incluir el concern en este caso)

Reply:
perfect!